plugins/scaffolder/src/components/ActionsPage/ActionsPage.tsx

```diff
@@ -67,6 +76,29 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+const examplesColumns: TableColumn<ActionExample>[] = [
+  {
+    title: 'Description',
+    field: 'description',
+    width: '20%',
+  },
+  {
+    title: 'Example',
+    render: ({ example }) => {
+      return (
+        <Grid>
```

Comment:
Does this wrapper really help? It looks lonely. Should it be removed or replaced with like `<Box padding={1}>` if that's what was intended?

Reply:
I'll also mention that maybe it would have been better to use a grid than a table. Then you could have made it responsive to different screen sizes, such that the description could be above the example on a narrow screen but on the side on desktops.

Reply:
I switched the grid item to a box and I swapped the table out for a grid.